test/unit/test_quantum_computer.py

```diff
@@ -431,11 +411,15 @@ def test_qc(client_configuration: QCSClientConfiguration):
 
 def test_qc_run(client_configuration: QCSClientConfiguration):
     qc = get_qc("9q-square-noisy-qvm", client_configuration=client_configuration)
-    bs = qc.run(qc.compile(Program(
-        Declare("ro", "BIT", 1),
-        X(0),
-        MEASURE(0, ("ro", 0)),
-    ).wrap_in_numshots_loop(3)))
+    bs = qc.run(
```

Comment:
How does this work without tacking on `.readout_data["ro"]`?

Reply:
It's there, on L425